src/main/java/com/betadevels/onlineshopping/payload/request/subscription/UpdateSubscriptionStatusRequest.java

```diff
@@ -0,0 +1,17 @@
+package com.betadevels.onlineshopping.payload.request.subscription;
+
+import com.betadevels.onlineshopping.enumerations.SubscriptionStatus;
+import io.dropwizard.jackson.JsonSnakeCase;
+import lombok.Getter;
+
+import java.util.List;
+
+/**
+ * Created by girijagodbole on 4/16/17.
+ */
+@JsonSnakeCase
+@Getter
+public class UpdateSubscriptionStatusRequest {
+    private List<Long> subscriptionIds;
+    private SubscriptionStatus status;
```

Comment:
If you are getting a single status for a list of subscriptions, how do you know if all the subscriptions are in the right state for them to be updated to the target state?